src/core.js

```diff
@@ -426,6 +426,19 @@
     iframe.style = "width: 100%; height: 100%";
     containerDiv.appendChild(iframe);
 
+    // Let CCP know if iframe is visible
+    iframe.onload = setTimeout(function() {
```

Comment:
Can we follow the structure of other iframe messages? i.e.
```
{event: eventName, data: data}
```

Reply:
i decided to use the iframe conduit since that would be cleaner